manifests/init.pp

```diff
@@ -123,6 +126,7 @@
   Optional[String[1]] $proxy_server                        = undef,
   Boolean $manage_package                                  = true,
   Enum['rsa', 'ed25519'] $ssh_key_type                     = 'ed25519',
+  Hash[String,String] $backuptime                          = { 'default' => '18:30:00' },
```

Comment:
We can also update this a bit I guess:
```suggestion
  Hash[String[1],String[1]] $backuptime                          = { 'default' => '18:30:00' },
```

This won't prohibit broken configs, but at least prevents empty strings